src/main/java/com/worldpay/sdk/Http.java

```diff
@@ -203,32 +207,31 @@ private HttpURLConnection createRequest(RequestMethod method, String uri, Object
         String fullUri = baseUri + uri;
         HttpURLConnection httpURLConnection = HttpUrlConnection.getConnection(fullUri);
         try {
-            httpURLConnection.setRequestProperty(WorldpayLibraryConstants.AUTHORIZATION, serviceKey);
-            final String propertiesWithVersion =
-                systemProperties.concat(WorldpayLibraryConstants.COMMA).concat(getVersion());
-            httpURLConnection.setRequestProperty(WorldpayLibraryConstants.WP_CLIENT_USER_AGENT, propertiesWithVersion);
+            httpURLConnection.setRequestProperty(AUTHORIZATION, serviceKey);
+            final String propertiesWithVersion = systemProperties.concat(getVersion());
+            httpURLConnection.setRequestProperty(WP_CLIENT_USER_AGENT, propertiesWithVersion);
 
             DataOutputStream dataOutputStream;
             switch (method) {
                 case GET:
-                    httpURLConnection.setRequestMethod(WorldpayLibraryConstants.GET);
+                    httpURLConnection.setRequestMethod(GET);
                     break;
                 case POST:
-                    httpURLConnection.setRequestMethod(WorldpayLibraryConstants.POST);
+                    httpURLConnection.setRequestMethod(POST);
                     dataOutputStream = new DataOutputStream(httpURLConnection.getOutputStream());
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) com.worldpay.sdk.Http.createRequest(Http$RequestMethod, String, Object) may fail to close stream [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.worldpay.io:9000/coding_rules#rule_key=findbugs%3AOS_OPEN_STREAM)